proto/pdpb.proto

```diff
@@ -130,6 +131,17 @@ message AskSplitResponse {
     
 }
 
+message GetAllMetaRequest {
+    // support getting all nodes and stores
+    optional MetaType meta_type = 1;
+}
+
+message GetAllMetaResponse {
```

Comment:
No need, pd doesn't know where the leader is, so it is the client's responsibility to get it.
Client only needs to care to cache all node's address first, and get the region with key, then it will communicate with raft server directly.